docker-compose.override.yml

```diff
@@ -61,6 +61,8 @@ services:
     command: init
     volumes:
       - ./openverse_catalog:/usr/local/airflow/openverse_catalog
+    stdin_open: true
+    tty: true
```

Comment:
I guess not. Without them you couldn't use pdb with dc up though on running DAGs. I'd rather leave it so that it just works for what ever poor soul ends up trying that in the future. It's just in the local config so it doesn't hurt.